content/ru/event-streaming.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Потоковая передача событий
+status: Completed
+category: concept
+tags: ["methodology", "networking", ""]
+---
+
+Потоковая передача событий — это подход, при котором одно приложение отправляет данные о событиях в другие, чтобы постоянно сообщать им о том, что оно делает.
+Представьте себе службу, транслирующую все, что она делает, всем остальным службам.
+Каждое действие, выполняемое такой службой, называется событием, отсюда и потоковая передача событий.
+Например, Служба автоматизированных котировок Национальной ассоциации дилеров по ценным бумагам (NASDAQ) каждую секунду публикует свежую информацию о ценах на акции и сырьё.
+Если бы у вас было приложение, отслеживающее стоимость определенного набора акций, вы бы хотели получать обновлённую информацию практически в режиме реального времени.
+Yahoo! Finance предоставляет [API](/ru/application-programming-interface/), который извлекает данные из NASDAQ и отправляет (или передает) информацию (или события) в любое приложение, которое на него подписывается.
```

Comment:
This link won't work at the moment since we haven't translated this page into Russian yet. We'll need to revert when another localisation is ready.
```suggestion
Yahoo! Finance предоставляет [API](/application-programming-interface/), который извлекает данные из NASDAQ и отправляет (или передает) информацию (или события) в любое приложение, которое на него подписывается.
```